README.md

```diff
@@ -111,23 +111,43 @@ Next / interim revision example:
 
 The generated `.vsix` will be under folder `./dist`
 
-## Publishing vscode-builtin-extensions to open-vsx
+## Packaging built-in vscode extensions in an extension-pack '.vsix` file.
```

Comment:
I think it's implied that an extension pack is itself a .vsix file, so we could omit  the last part.

Reply:
e.g. go with "## Packaging built-in vscode extensions in an extension-pack"

Reply:
I have updated it as suggested in two places. thanks Marc.